extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -586,9 +586,6 @@ export class AmpAdXOriginIframeHandler {
    * @param {boolean} inViewport
    */
   viewportCallback(inViewport) {
-    if (this.intersectionObserverHost_) {
```

Comment:
likely the removal of this code might have caused issue: https://github.com/ampproject/amphtml/issues/29759
@dvoytenko was it intended to remove this?